blurb/blurb.py

```diff
@@ -486,6 +486,9 @@ def finish_entry():
                 # we'll complain about the *first* error
                 # we see in the blurb file, which is a
                 # better user experience.
+                if key == "gh-issue" and int(value) < 32426:
```

Comment:
Let's put the magic number in a variable with a clear name?

Something like: `lowest_possible_issue_number`? I don't feel strongly about the bikeshed color, as long as the name conveys something about why we're using this specific magic number.